extensions/amp-analytics/0.1/vendors.js

```diff
@@ -2095,6 +2095,75 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'oracleInfinityAnalytics': {
+    'transport': {
+      'beacon': false,
```

Comment:
Are you sure you don't want to use these? They will fallback to image if not supported.

Reply:
No this is fine, we just recommend beacon if possible because it can be a better user experience.

Reply:
makes sense; however, our current endpoint is not currently accepting POST.  We'll refactor when that story has been approved and resourced internally.  Is there anything else we need to do for this to become a candidate for the next release?

Reply:
Sounds good. Nope this looks good. Should make it into our canary cut on Tuesday. Thanks for contributing!